src/components/HelpDrawer/HelpContent/SpectralProfilerHelpContent.tsx

```diff
@@ -9,78 +9,76 @@ export const SPECTRAL_PROFILER_HELP_CONTENT = (
                 statistic.
             </li>
             <li>
-                When the <code>Statistic</code> checkbox is selected: spectra with <em>different statistical quantities</em> from the selected region of the selected image (and the selected polarization if applicable) are displayed.
+                When the <b>Statistic</b> checkbox is selected: spectra with <em>different statistical quantities</em> from the selected region of the selected image (and the selected polarization if applicable) are displayed.
             </li>
             <li>
-                When the <code>Polarization</code> checkbox is selected: spectra with <em>different polarization components</em> from the selected image, the selected region and the selected statistic are displayed.
+                When the <b>Polarization</b> checkbox is selected: spectra with <em>different polarization components</em> from the selected image, the selected region and the selected statistic are displayed.
             </li>
         </ul>
-        <p>In short, if multiple spectra are plotted, only one option (Image, Region, Statistic or Polarization) can be varied at a time. All other options are fixed to a single value.</p>
-        <p>The cursor information of each profile is displayed at the bottom-left corner. The cursor information box can be resized.</p>
-
-        <h4>LIMITATION</h4>
         <p>
-            If the intensity units of different matched cubes are different (e.g., Jy/beam vs Kelvin), <em>no</em> unit conversion is applied in the multi-profile plot. Intensity unit conversion is only available with a <em>single</em>{" "}
-            profile with the <code>Conversion</code> tab of the spectral profiler settings dialog (the <code>cog</code> button at the top right corner of the widget).
+            In short, if multiple spectra are plotted, only one option (<b>Image</b>, <b>Region</b>, <b>Statistic</b> or <b>Polarization</b>) can be varied at a time. All other options are fixed to a single value.
         </p>
+        <p>The cursor information of each profile is displayed in the bottom-left corner. The cursor information field can be resized.</p>
 
         <h3 id="images">Image dropdown menu</h3>
         <p>
-            The image dropdown menu defaults to &quot;Active&quot; image which means the current image in the image viewer if it is in single-panel mode. If it is in multi-panel mode, the active image is highlighted with a red box. You may
-            use the animator widget or the image list widget to change the active image.
+            The <b>Image</b> dropdown menu defaults to &quot;Active&quot; image which means the current image in the Image Viewer if it is in single-panel mode. If it is in multi-panel mode, the active image is highlighted with a red box.
```

Comment:
`The <b>Image</b> dropdown menu defaults to "Active", which means the currently selected image. This is the image which is visible in the Image Viewer (if it is in the single-panel mode). If the viewer is in the multi-panel mode, the active image is highlighted with a red box.`

(Copied from the same description elsewhere.)